lib/jsonapi/relationship.rb

```diff
@@ -70,6 +93,12 @@ def initialize(name, options = {})
         @foreign_key_on = options.fetch(:foreign_key_on, :self)
       end
 
+      def to_s
+        # :nocov:
+        "#{parent_resource}.#{name}(#{belongs_to? ? 'ToOne' : 'BelongsToOne'})"
```

Comment:
```suggestion
        "#{parent_resource}.#{name}(#{belongs_to? ? 'BelongsToOne' : 'ToOne'})"
```

Not sure why we want to distinguish `ToOne` from `BelongsToOne` at the resource relationship level. But I think the ternary was backwards.